resources/scripts/components/dashboard/AccountOverviewContainer.tsx

```diff
@@ -1,17 +1,25 @@
 import { useLocation } from 'react-router-dom';
+import { State, useStoreState } from 'easy-peasy';
+import type { ApplicationStore } from '@/state';
+import sha256 from 'crypto-js/sha256';
 
 import MessageBox from '@/components/MessageBox';
 import ConfigureTwoFactorForm from '@/components/dashboard/forms/ConfigureTwoFactorForm';
 import UpdateEmailAddressForm from '@/components/dashboard/forms/UpdateEmailAddressForm';
 import UpdatePasswordForm from '@/components/dashboard/forms/UpdatePasswordForm';
 import ContentBox from '@/components/elements/ContentBox';
 import PageContentBlock from '@/components/elements/PageContentBlock';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/elements/Tabs';
 
 import Code from '../elements/Code';
 
 export default () => {
     const { state } = useLocation();
 
+    const user = useStoreState((state: State<ApplicationStore>) => state.user.data!);
+    const emailHash = sha256(user.email).toString();
+    const avatar = `https://www.gravatar.com/avatar/${emailHash}?s=128&d=identicon`;
```

Comment:
no

Reply:
how to turn off gravatar on my instance of pyro

Reply:
gravatar is for when you don't want to deal with uploadable profile pictures
i do not want to deal with uploadable profile pictures

Reply:
i do want to deal with uploadable pictures so how turn it offs

Reply:
> > gravatar is for when you don't want to deal with uploadable profile pictures
> > i do not want to deal with uploadable profile pictures
> 
> Real

REMUV GAYVATAR RIGHT NOW OR ELSE I NUKE CHINA